Makefile

```diff
@@ -87,4 +87,7 @@ e2e:
 e2e-prod:
 	FOCUS="\[Real\]" ./hack/e2e.sh
 
+e2e-scaleupdown:
+	FOCUS="\[ScaleUpDown\] \[Real\]" TIMEOUT=30m ./hack/e2e.sh
```

Comment:
nit: remove the space between the tags

Reply:
The space is intentional. This make target should match `[ScaleUpDown] [Real]` exactly. It would be cleaner to just specify `[ScaleUpDown]` for this target but part of the changes in this pr involves making sure azure test clients check for either the Fake or Real focus.